portfolio/src/main/webapp/style.css

```diff
@@ -1,52 +1,60 @@
-.menu ul {
+*{
```

Comment:
This matches every element on the page, including the root html and body elements; if you just wanted to change one of those, I'd make this rule more specific. If you want all elements to have these settings, that's fine too.